src/openforms/scss/components/admin/_servicefetchconfiguration_form.scss

```diff
@@ -0,0 +1,3 @@
+.servicefetchconfiguration-form {
+  margin-top: 2em;
+}
```

Comment:
components should never affect other components! this is a top-level block, so in the CSS box model you are only allowed to touch borders and paddings. Margins push into other siblings which leads to unpredictable layouts.

Instead, the parent container should probably be adapted with flexbox gaps or better spacing logic (possibly through a modifier).

Reply:
replaced this with flexbox gaps on the `servicefetchconfiguration-picker` element